python/semantic_kernel/contents/streaming_chat_message_content.py

```diff
@@ -68,3 +72,31 @@ def __add__(self, other: "StreamingChatMessageContent") -> "StreamingChatMessage
             encoding=self.encoding,
             finish_reason=self.finish_reason or other.finish_reason,
         )
+
+    def to_prompt(self, root_key: str) -> str:
+        """Convert the ChatMessageContent to a prompt.
+
+        Returns:
+            str - The prompt from the ChatMessageContent.
+        """
+
+        root = Element(root_key)
+        root.set("role", self.role.value)
+        root.set("metadata", json.dumps(self.metadata))
```

Comment:
Same question. Remove this and add an attribute for tool call as part of this class?